modules/insticatorBidAdapter.js

```diff
@@ -60,17 +66,19 @@ function buildImpression(bidRequest) {
     });
   }
 
+  const gpid = deepAccess(bidRequest, 'ortb2Imp.ext.data.pbadslot');
```

Comment:
look for gpid and fallback to this; see #7367

Reply:
Now use ortb2Imp.ext.gpid